### DISCUSSION TITLE
Using IBIS with Databricks

### DISCUSSION THREAD

**Discussion by nikhilmakan02** [ibis-project/ibis#5510] [Q&A] [answered]
Hi Team, just wondering if you can use Ibis with a Databricks as a backend given its Spark? I guess if not am I correct in saying given that you just pass in a SQL string to the Databricks SQL Connector in Python, you could use Ibis to compile the SQL? But just handle the connection and execute part yourself using the Databricks SQL Connector?

**Comment by gforsyth** [comment, marked as answer]
Hey @nikhilmakan02 -- sorry we missed this!

I haven't used databricks much, so I'm not entirely sure how to do this, but so long as you can get a pyspark session going, you can then use `ibis` to interact with it, something like:

```python
>>> import ibis
>>> from pyspark.sql import SparkSession
>>> session = SparkSession.builder.getOrCreate()  # <-- thie line might be different on databricks?
>>> ibis.pyspark.connect(session)
```

**Comment by nikhilmakan02** [comment]
Thanks Gil, appreciate the response on this. I will see how I can
incorporate that.

Kind regards
Nikhil Makan
